### ISSUE TITLE
Smaller Icons with CSS

### ISSUE THREAD

**Issue by LDigital84** [FortAwesome/Font-Awesome#4375]
I know you can enlarge an icon using `fa-lg` or the other enlarge classes. I would love to see some reduction classes that allow me to easily reduce the size of an icon. 

For instance you could add a `fa-rd` (rd = reduce) for a decrease in size. 

I ask because recently I ran into situations where I wanted to reduce the icon size but not the font size. I manual was able to do this but having this baked into Font Awesome would be, well, awesome!

**Comment by dimo414** [comment]
Agreed, I just ran into a use case where I wanted an icon a little smaller.

**Comment by Shriti507** [comment]
Font Awesome already provides utilities for size reduction, like 
- fa-xs
- fa-2xs
These utilities reduce the size of the icons without affecting the text size, effectively addressing the issue.

**Comment by sensibleworld** [comment]
Thanks for opening this issue! Since it’s been a while and things have changed quite a bit in Font Awesome since this was originally posted, we’re closing it out to help keep our issue tracker focused on current work.

If this is still important for your project, please feel free to re-open or file a fresh issue with updated details. Thanks so much!
